extensions/bmrks/src/search-bookmarks.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { User } from "@supabase/supabase-js";
+import { Action, ActionPanel, Detail, Icon, List, openExtensionPreferences } from "@raycast/api";
+import { useAuth } from "../lib/use-auth";
+import { useGroups } from "../lib/use-groups";
+import { useBookmarks } from "../lib/use-bookmarks";
+import { getHostname } from "../lib/getHostname";
+
+export function SearchBookmarks({ user }: { user: User }) {
+  const { data: groups, isLoading: isLoadingGroups } = useGroups(user);
+  const [groupId, setGroupId] = React.useState<string | undefined>("");
+  const { data: bookmarks, isLoading: isLoadingBookmarks } = useBookmarks(groupId);
+
+  React.useEffect(() => {
+    if (groups.length > 0) {
+      setGroupId(groups[0].id);
+    }
+  }, [groups]);
+
+  return (
+    <List
+      isLoading={isLoadingGroups || isLoadingBookmarks}
+      searchBarAccessory={
+        <List.Dropdown tooltip="Groups" value={groupId} onChange={setGroupId}>
+          {groups.map((group) => {
+            return <List.Dropdown.Item key={group.id} title={group.name || ""} value={group.id} />;
+          })}
+        </List.Dropdown>
+      }
+    >
+      {bookmarks.length > 0 &&
+        bookmarks.map((bookmark) => {
+          const hostname = bookmark.type === "link" ? getHostname(bookmark.url || "") : "";
+          let icon;
+          if (bookmark.favicon_url) {
+            icon = bookmark.favicon_url;
+          }
+          if (bookmark.type === "color") {
+            icon = { source: Icon.CircleFilled, tintColor: bookmark.title };
```

Comment:
Maybe add `adjustContrast: false` so it doesn't change the color

Reply:
Good shout 👌